### DISCUSSION TITLE
python controls GUI frequency limits

### DISCUSSION THREAD

**Discussion by dwkmff** [jankae/LibreVNA#235] [General]
When using python to control the GUI, the commands VNA:FREQuency:START or VNA:FREQuency:STOP cannot be set to 5G.
The python file witch named retrieve_trace_data.py provided by Jan.
![image](https://github.com/jankae/LibreVNA/assets/155036518/588359b5-c986-4606-8a65-28beb85a62fe)

**Comment by dwkmff** [comment]
that via SCPI the frequency can not be set above 4294967295 Hz (which is exactly max. value of uint32). When I send command ":VNA:FREQ:STOP 4294967295" it works, but ":VNA:FREQ:STOP 4294967296" or any higher value results in "ERROR" response. I am using current release (1.2.1).

**Comment by jankae** [comment]
I can't test this myself right now but I can't find any obvious mistake in the code. I will check this in a couple of days, should be an easy fix.

In any case, 1.2.1 is not the latest release. Please check if the bug is already  fixed in 1.5.0

**Reply by dwkmff** [reply]
Thank you very much for your prompt response. I will first test whether this command can be successfully executed on version 1.5.0.
